frontend/src/RecordingPageComp/RecordingComp.jsx

```diff
@@ -70,18 +70,24 @@ function RecordingComp() {
     audio.src = audioSrc;
   }
 
+  useEffect(() => {
+    if (recorderControls.recordingTime === 10) recorderControls.stopRecording();
```

Comment:
Not sure if recordingTime is a float, but if so, I would use a '>= 10' or '> 10' here to catch edge case where recordingTime may not exactly equal 10, given it is updated every second